### ISSUE TITLE
add script to print all hardware and software versions

### ISSUE THREAD

**Issue by anuejn** [apertus-open-source-cinema/axiom-firmware#147]
to ease (remote) debugging. something like axiom-info might be a good name

**Comment by rroohhh** [comment]
Maybe it is time to switch to nix and let it handle all that for us? :)

**Comment by anuejn** [comment]
Well i was more thinking about the hardware and the specific firmware running on the hardwaee

**Comment by rroohhh** [comment]
Yeah I understand, atleast the software versions would be trivial with nix, and hardware versions would probably live in the nix config aswell?

**Comment by herbertp** [comment]
Not sure how 'nix' would figure out the hardware versions/revisions though :)
